cmake/modules/hunter_get_lang_standard_flag.cmake

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2017 Pawel Bylica
+# All rights reserved.
+
+include(hunter_status_debug)
+
+function(hunter_get_lang_standard_flag LANG OUTPUT)
```

Comment:
Also I think for debugging purpose we can print next variables using `hunter_status_debug`:
* CMAKE_${LANG}_STANDARD_DEFAULT
* CMAKE_${LANG}_STANDARD
* CMAKE_${LANG}_EXTENSIONS
* CMAKE_${LANG}_STANDARD_REQUIRED